test/e2e/specs/constants.js

```diff
@@ -1,11 +1,11 @@
 import path from 'path';
 
 export const TEST_IMAGE_PATH = path.normalize(
-	path.join( __dirname, '..', 'image-uploads', 'image0.jpg' )
+	path.join( __dirname, '..', 'image-uploads', 'test-image-01.png' )
```

Comment:
For the record, `path.join()` calls `path.normalize()` internally, so this is redundant.

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/58752/commits/5ee9718561dc0f3b273cbfb283b086b2e0206e97